packages/ibm-products/src/components/Card/Card.tsx

```diff
@@ -190,8 +199,10 @@ export const Card = forwardRef(
         );
       }
 
-      const icons = getIcons().map(
-        ({ id, icon: Icon, onClick, iconDescription, href, ...rest }) => {
+      const icons = iconList?.map(
+        ({ id, icon: Icon, onClick, iconDescription, href, link, ...rest }) => {
+          const { url, ...linkProps } = link ?? {};
```

Comment:
```suggestion
          const { url, ...linkProps } = link ?? {url: href};
```

Means that on line 230 you don't need the extra `||`

Reply:
Good suggestion. I have pushed the changes.